### ISSUE TITLE
cmd/go, misc/cgo: clang does not generate .edata sections

### ISSUE THREAD

**Issue by zx2c4** [golang/go#46719]
We're currently using the .edata section to check whether cgo generated a PE binary with the expected symbols. While binutils generates this section, clang's LLD does not. It still exports the symbols, but just doesn't have the .edata section. This issue is to track the (eventual) refactoring of those tests.

**Comment by zx2c4** [comment]
For now, CL 327549 just skips them.

**Comment by dmitshur** [comment]
Is this related to issue #46510?

**Comment by ianlancetaylor** [comment]
To clarify, this is only related to tests.

**Comment by zx2c4** [comment]
> Is this related to issue #46510?

Yes, same thing basically. There's one other test that uses .edata in addition to the one mentioned in that issue. Both changed in https://github.com/golang/go/commit/24cff0f0444793be81062684c478a3f7ca955499
